Optimizely-Kit.js

```diff
@@ -0,0 +1,589 @@
+(function (global, factory) {
```

Comment:
This should be renamed to Optimizely.js in the root directory, rather than copied directly from build/Optimizely-Kit.js because it's Optimizely.js in the database.